constraints.txt

```diff
@@ -0,0 +1,4 @@
+# Constraints that apply to requirements-test.txt
```

Comment:
If we're documenting that `constraints.txt` (only) applies to `requirements-test.txt`, should we call it `constraints-test.txt`?

Reply:
If we need constraints on the base requirements in the future, I'd rather use the same file, so I'll change the comment to be less specific.